pkg/vere/disk.c

```diff
@@ -882,10 +884,35 @@ u3_disk_init(c3_c* pax_c, u3_disk_cb cb_u)
       return 0;
     }
 
-    //  Arbitrarily choosing 1TB as a "large enough" mapsize
+    //  validation before migration
     //
-    //  per the LMDB docs:
-    //  "[..] on 64-bit there is no penalty for making this huge (say 1TB)."
+
+    //  migrate to the correct disk format
+    //
+    u3_disk_migrate(log_u);
+
+    //  find the latest epoch directory
+    //  XX separate function?
```

Comment:
yeah I think this deserves its own function, even if it's only called once -- it's conceptually important enough to live on its own, and I'd say it's likely to be used by other tools too.